pygenesys/model_info.py

```diff
@@ -38,28 +41,36 @@ def __init__(self,
         N_seasons : int
             The number of seasons in the simulation. Several values
             are acceptable. E.g.
-                * 1; there are no seasonal differences
+                * 1; there are no seasonal differences (not implemented)
                 * 4; spring,summer,fall,winter
                 * 365; full year, daily resolution
         N_hours : int
             The number of hours in the simulation. Several values
             are acceptable. E.g.
-                * 1; there is no daily variation
-                * 2; diurnal variation, step function
+                * 1; there is no daily variation  (not implemented)
```

Comment:
```suggestion
                * 1; there is daily variation  (not implemented)
```

Reply:
N_hours is the number of hours modeled in the simulation. If you put 24, there are 24 hours in a day. If you put 2, there is a day/night variation. If you put 1, it is a flat line. I wrote "not implemented" because it's not supported yet.

Reply:
Oh, ok. That makes sense!